packages/opentelemetry-instrumentation/README.md

```diff
@@ -0,0 +1,153 @@
+# OpenTelemetry Instrumentation for web and node
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation
+```
+
+## Usage in Node
+
+```typescript
+import {
+  Instrumentation,
+  InstrumentationNodeModuleDefinition,
+  InstrumentationNodeModuleFile,
+} from '@opentelemetry/instrumentation';
+
+import * as module_name_to_be_pached from 'module_name_to_be_pached';
```

Comment:
It is important to use `import type` to prevent a `require` from being emitted and causing a circular dependency.

```suggestion
import type * as module_name_to_be_pached from 'module_name_to_be_pached';
```